admin/cache-mp-gcc.cmake

```diff
@@ -0,0 +1,9 @@
+# Cache settings for building the macOS release with GCC and OpenMP
```

Comment:
What about including these lines in the `admin/build-release.sh` script, so that we don't have to maintain too many files for releasing GMT.
```
cat > cache-mp-gcc.cmake << EOF
SET ( CMAKE_C_COMPILER "/opt/local/bin/gcc-mp-9" CACHE STRING "GNU MP C compiler" )
SET ( CMAKE_CXX_COMPILER "/opt/local/bin/g++-mp-9" CACHE STRING "GNU MP C++ compiler" )
SET ( CMAKE_C_FLAGS -flax-vector-conversions CACHE STRING "C FLAGS")
SET ( CMAKE_C_FLAGS_DEBUG -flax-vector-conversions CACHE STRING "C FLAGS DEBUG")
SET ( CMAKE_C_FLAGS_RELEASE -flax-vector-conversions CACHE STRING "C FLAGS RELEASE")
SET ( OpenMP_C_FLAGS -flax-vector-conversions CACHE STRING "C FLAGS OPENMP")
EOF
```

Reply:
OK, done.